narwhal/types/proto/narwhal.proto

```diff
@@ -5,152 +5,13 @@ syntax = "proto3";
 
 package narwhal;
 
-message CertificateDigest {
-    bytes digest = 1;
-}
-
 message Transaction {
     bytes transaction = 1;
 }
```

Comment:
I wonder if we could simplify this interface now that grpc isn't used elsewhere...

Reply:
Yeah we can remove Sui -> Narwhal worker RPC interface easily in future, if everyone agrees.